invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/accordion.overrides

```diff
@@ -3,6 +3,15 @@
   &.tab-menu-accordion {
     transform: none;
   }
+
+  transform: rotate(90deg);
+}
+
+.ui.accordion .title .icon.icon:not(.button) {
```

Comment:
See also the default [implementation](https://github.com/Semantic-Org/Semantic-UI/blob/597843ab84d565cccfd8fd7719416350ae73e734/src/definitions/modules/accordion.less#L65) in Semantic UI. I guess the default behaviour broke because we overrid the less rules?